test/library/packages/Yaml/percent-q.future

```diff
@@ -0,0 +1 @@
+yaml deserializer isn't properly invoked by '%?' format specifier
```

Comment:
is there a github issue that could be referenced here?

Reply:
Not yet, but I'll make one